PWGCF/FemtoDream/Tasks/femtoDreamPairTaskTrackHF.cxx

```diff
@@ -0,0 +1,376 @@
+// Copyright 2019-2022 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file femtoDreamPairTaskTrackHF.cxx
+/// \brief Tasks that reads the track tables used for the pairing and builds pairs of two tracks
+/// \author Ravindra SIngh, GSI, ravindra.singh@cern.ch
+
+#include <Framework/Expressions.h>
+#include <sys/stat.h>
+#include <cstdint>
+#include <vector>
+#include "Framework/AnalysisTask.h"
```

Comment:
```suggestion
#include <sys/stat.h> // Why?
#include <cstdint> // Why?
#include <vector>

#include "Framework/Expressions.h"
#include "Framework/AnalysisTask.h"
```

Reply:
Hi @vkucera, I was using a reference from one existing task, and due to my oversight I didn't excluded these headers. I will remove them in version

Reply:
HI @vkucera, I have implemented all the comments. Please have  a look and let me know if all are okay

Reply:
Looks good, thanks.